src/injections/withPostFailure/README.md

```diff
@@ -0,0 +1,32 @@
+## withPostFailure
+
+The `withPostFailure` injector allows to inject behaviour after the service call fails and the `onFailure` effect is executed. This means that the `${action.target}Loading` and `${action.target}Error` are set before `withPostFailure` is called.
```

Comment:
I'd use `allows us` or `allows you`

Reply:
I don't think `us` or `you` should be used while writing documentation. Best practices in formal writing recommend avoiding these. https://www2.ivcc.edu/rambo/tip_formal_writing_voice.htm